lib/ecs_deploy/scheduled_task.rb

```diff
@@ -70,6 +71,10 @@ def put_targets
           task_count: @task_count,
         },
       }
+      if @network_configuration
+        target[:ecs_parameters].merge!(network_configuration: @network_configuration)
```

Comment:
👍 
https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/CloudWatchEvents/Types/EcsParameters.html

By the way, it would be better we support also `launch_type`.

Reply:
OK. I will submit another PR.